middlewares/authentication.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+const { api } = require('actionhero')
+
+const MAX_AGE = 1000 * 60 * 5 // max age of signed message is 5m
+
+/**
+ * verify auth headers; example for creating a signed auth header:
+ * web3.eth.accounts.sign(`${accountId}|${Date.now()}`, `0x${privateKey}`)
```

Comment:
Should we adjust the comment? :D
```suggestion
 * web3.eth.accounts.sign(`${Date.now()}`, `0x${privateKey}`)
```